common/tier4_vehicle_rviz_plugin/src/tools/turn_signal.cpp

```diff
@@ -95,10 +95,9 @@ void TurnSignalDisplay::update(float wall_dt, float ros_dt)
   unsigned int signal_type;
```

Comment:
@1222-takeshi 
I think you need to initialize this value with default value

Reply:
Thank you for your review :smile: 
I fixed https://github.com/autowarefoundation/autoware.universe/pull/1379/commits/66246fc205347aff6d2b2ffd2a059d5189eb87e8